pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/ServerRequestPlanVisitor.java

```diff
@@ -90,6 +92,9 @@ public static ServerPlanRequestContext build(MailboxService<TransferableBlock> m
     PinotQuery pinotQuery = new PinotQuery();
     pinotQuery.setLimit(DEFAULT_LEAF_NODE_LIMIT);
     pinotQuery.setExplain(false);
+    pinotQuery.setQueryOptions(ImmutableMap.of(
+        CommonConstants.Broker.Request.QueryOptionKey.SERVER_RETURN_FINAL_RESULT,
```

Comment:
If we want to re-use the code, let's use a different config option that is invisible to user.

Reply:
I removed this flag altogether.